microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -95,3 +104,5 @@ For any WebView2 app, make sure to follow our recommended WebView2 security best
 [WebView2ProcessFailedEvent]: /microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs "ICoreWebView2ProcessFailedEventArgs | Microsoft Docs"
 
 [MicrosoftedgeinsiderDownload]: https://www.microsoftedgeinsider.com/download "Download Microsoft Edge Insider Channels"
+
+[CreateCoreWebView2Environment]: /microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.961.33#createcorewebview2environment "CreateCoreWebView2Environment | Microsoft Docs"
```

Comment:
```suggestion
[CreateCoreWebView2Environment]: /microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environment "CreateCoreWebView2Environment | Microsoft Docs"
```
Doc build says:
Line 76, Column 44: [Suggestion-preserve-view-not-set] Did you mean to link to a specific version? It's usually better not to specify a version, so the link always goes to the canonical version. If you really need to link to a specific version, add &preserve-view=true to the URL: /microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.961.33#createcorewebview2environment. Otherwise, remove the view parameter.